cli/popper/commands/cmd_run.py

```diff
@@ -54,6 +55,31 @@ def cli(ctx, pipeline, timeout, skip, ignore_errors):
                 "Run popper init --help for more info.", fg='yellow')
         sys.exit(0)
 
+    # Checks if popper is being executed by Travis.
+    if 'TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true':
+        # Get last commit message
+        commit = os.environ['TRAVIS_COMMIT_MESSAGE']
+
+        if "popperci:skip" in commit:
+            pu.info("popperci:skip flag detected. "
+                    "Skipping execution of commit")
+            sys.exit(0)
+
+        if "popperci:whitelist" in commit:
+            pu.info("popperci:whitelist flag detected.")
+            try:
+                # Checks if the last commit message has the flag
+                # `popperci:whitelist[pipeline]` and gets the pipeline.
+                pipeline = re.search('popperci:whitelist\[(.+?)\]',
+                                     commit).group(1)
+                pu.info("Executing popperci:whitelist[{}]"
+                        .format(pipeline))
+            except AttributeError:
+                pipeline = None
+                pu.warn("Couldn't find pipeline associated with the "
+                        "popperci:whitelist flag. "
+                        "Assigning pipeline to None")
```

Comment:
this should be independent of a CI service. So we need to use instead `git` to look for the commit message, e.g. to run something like `git log -1 --pretty=%B` and store that on the string and parse it. Also, we can rename `popperci:` to `popper` (that `popperci` is legacy from the golang days)